bundles/org.openhab.binding.fmiweather/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,382 @@
+				<option value="62">Rain, not freezing, moderate</option>
+				<option value="63">Rain, not freezing, heavy</option>
+				<option value="64">Rain, freezing, slight</option>
+				<option value="65">Rain, freezing, moderate</option>
+				<option value="66">Rain, freezing, heavy</option>
+				<option value="67">Rain (or drizzle) and snow, slight</option>
+				<option value="68">Rain (or drizzle) and snow, moderate or heavy</option>
+				<option value="69">Reserved</option>
+				<option value="70">SNOW</option>
+				<option value="71">Snow, slight</option>
+				<option value="72">Snow, moderate</option>
+				<option value="73">Snow, heavy</option>
+				<option value="74">Ice pellets, slight</option>
+				<option value="75">Ice pellets, moderate</option>
+				<option value="76">Ice pellets, heavy</option>
+				<option value="77">Snow grains</option>
+				<option value="78">Ice crystals</option>
+				<option value="79">Reserved</option>
+				<option value="80">Shower(s) or intermittent precipitation</option>
+				<option value="81">Rain shower(s) or intermittent rain, slight</option>
+				<option value="82">Rain shower(s) or intermittent rain, moderate</option>
+				<option value="83">Rain shower(s) or intermittent rain, heavy</option>
+				<option value="84">Rain shower(s) or intermittent rain, violent</option>
+				<option value="85">Snow shower(s) or intermittent snow, slight</option>
+				<option value="86">Snow shower(s) or intermittent snow, moderate</option>
+				<option value="87">Snow shower(s) or intermittent snow, heavy</option>
+				<option value="88">Reserved</option>
+				<option value="89">Hail</option>
+				<option value="90">Thunderstorm</option>
+				<option value="91">Thunderstorm, slight or moderate, with no precipitation</option>
+				<option value="92">Thunderstorm, slight or moderate, with rain showers and/or snow showers</option>
+				<option value="93">Thunderstorm, slight or moderate, with hail</option>
+				<option value="94">Thunderstorm, heavy, with no precipitation</option>
+				<option value="95">Thunderstorm, heavy, with rain showers and/or snow showers</option>
+				<option value="96">Thunderstorm, heavy, with hail</option>
+				<!-- 97-98: Reserved-->
+				<option value="99">Tornado</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="total-cloud-cover-channel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Total Cloud Cover</label>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="precipitation-intensity-channel">
+		<item-type>Number:Speed</item-type>
+		<label>Precipitation Intensity</label>
+		<description>Equivalent to the precipitation amount if the same intensity prevails for an hour. </description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	
+	<channel-type id="weather-id-channel">
+		<item-type>Number</item-type>
+		<label>Prevailing weather id</label>
+		<category>Pressure</category>
```

Comment:
Does the category match? A description might be helpful here as well.

Reply:
Fixed in 4d2aadb, please review if the description is OK